<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-PTI6DNB4" name="com.vTiger.CreateOrganizationTest.CreateDuplicateOrganizationTest" tests="1" failures="1" timestamp="04 Dec 2019 09:28:29 GMT" time="32.653" errors="0">
  <testcase name="cretaeDuplicate_OrgTest" time="32.653" classname="com.vTiger.CreateOrganizationTest.CreateDuplicateOrganizationTest">
    <failure type="java.lang.AssertionError" message="expected [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM] but found [Administrator - Organizations - vtiger CRM 5 - Commercial Open Source CRM]">
      <![CDATA[java.lang.AssertionError: expected [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM] but found [Administrator - Organizations - vtiger CRM 5 - Commercial Open Source CRM]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.vTiger.CreateOrganizationTest.CreateDuplicateOrganizationTest.cretaeDuplicate_OrgTest(CreateDuplicateOrganizationTest.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- cretaeDuplicate_OrgTest -->
</testsuite> <!-- com.vTiger.CreateOrganizationTest.CreateDuplicateOrganizationTest -->
